lib/graphql/get_app_by_api_key.graphql

```diff
@@ -0,0 +1,11 @@
+query getApp($api_key: String!) {
```

Comment:
You can put this in your own project type `lib/project_types/extension/graphql/get_app_by_api_key.graphql` if you are the only one using this query.

Reply:
👍  Moved.